logging-modules/splunk-with-log4j2/pom.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
```

Comment:
Hi @JavierPeris ,
I was getting conflicts in the build due to other loggers because there are other submodules in which they have other loggers already (other than log4j2).

Error for reference:
`SLF4J(W): Class path contains multiple SLF4J providers.
SLF4J(W): Found provider [org.apache.logging.slf4j.SLF4JServiceProvider@64711bf2]
SLF4J(W): Found provider [ch.qos.logback.classic.spi.LogbackServiceProvider@169da7f2]
SLF4J(W): See https://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J(I): Actual provider is of type [org.apache.logging.slf4j.SLF4JServiceProvider@64711bf2] `